aces/Loris
# [added function] Group error message cell positioning

## Brief summary of changes

When working with Loris Form, one can only position the error message to row level or to element level if this is a standalone element. 

If the items is in a big table, Loris doesn't support cell level error message positioning, though the table rendering is supported and the row level error message could somehow be supported. Because of the fact that here we have two interpretations for the same id (element vs table row), the idea is not quite clear because one needs to find the table row id instead of element id to be able to show positioning error message. this could create some confusion for new developers.

The modification here will add this capacity, with the following formats:

`$errors[<table id>] = ["<cell id>" => "<error message>"]`

or `$errors[<table id>] = ["<cell id 1>" => "<error message 1>", "<cell id 2>" => "<error message 2>"]`

I suppose that this could be an added capacity for form processing.

#### Testing instructions (if applicable)

1. Make a item like before, test error positioning capacity, in additional form validation instead of XIN rules.

2. Make a grouped item, with 1 and 2 items to verify the validation error message positioning capacity

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
N/A
